grafana-operator/grafana-operator
# Pull in notifier provisioning implementation from infoblox's fork

## Description


- Pulls in notification channel implementation from the infoblox fork https://github.com/infobloxopen/grafana-operator

## Relevant issues/tickets


## Type of change




## Checklist


## Verification steps

- Run the operator and apply the example notification channel from `deploy/examples/notificationchannels/`
- Log in as admin user through the UI and verify that once the grafana instance is up and reconciled the notification channel appears.
